CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+= Contributing to the Pacemaker project =
+
+If you find Pacemaker useful and wish to support the project, you can:
+
+* Spread the word (on blogs, social media, mailing lists, Q&A websites, etc.).
+* Join and participate in the [mailing lists](http://clusterlabs.org/mailman/listinfo/).
+* Report [bugs and new feature requests](http://bugs.clusterlabs.org/).
+* Contribute documentation, bug fixes, or features to the code base.
+
+If you would like to contribute code base changes, please read
+[Pacemaker Development](http://clusterlabs.org/doc/en-US/Pacemaker/1.1-pcs/html-single/Pacemaker_Development/index.html)
+for detailed information about pull requests and policies.
```

Comment:
I suppose/hope this will be overriden with `.github/CONTRIBUTING.md` as proposed in #1121.
Reason is simple, when one i already doing an act of contribution (note the intended tight interaction of the file and GitHub when enabling such an act), she doesn't want to be bothered with "how can I contribute" :-)